src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -990,6 +991,10 @@ class TaskBuilderTest extends UnitTest {
 
       assert(!taskInfo.hasCommand)
       assert(cmd.getValue == "chmod ug+rx '/custom/executor' && exec '/custom/executor' a b c")
+      assert(taskInfo.getExecutor.getResourcesList == Seq(
+        ScalarResource.cpus(0.1),
+        ScalarResource.memory(32.0)
+      ).map(resourceToProto).asJava)
```

Comment:
Test will be easier to read with the constant; `TaskBuilder.defaultExecutorResources`